### PR TITLE
Bump Ubuntu

### PR DESCRIPTION
**Pull request by titanous**
14.04.2 includes the backported Utopic 3.16 kernel.

### CODE REVIEW

File: util/packer/ubuntu-14.04/scripts/upgrade.sh [hunk 1]
```diff
@@ -15,11 +15,6 @@ if [[ "${PACKER_BUILDER_TYPE}" == "virtualbox-ovf" ]]; then
   apt-get remove --purge -y virtualbox-guest-dkms virtualbox-guest-utils virtualbox-guest-x11
 fi
 
-apt-get install --install-recommends linux-generic-lts-utopic \
-  -y \
-  -o Dpkg::Options::="--force-confdef" \
-  -o Dpkg::Options::="--force-confold"
-
```
Comment (lmars) [inline util/packer/ubuntu-14.04/scripts/upgrade.sh]: Could we now drop this whole file and just use the provided kernel?

File: script/release-vm-images [hunk 2]
```diff
@@ -34,9 +34,9 @@ source "${ROOT}/script/lib/aws.sh"
 source "${ROOT}/script/lib/util.sh"
 
 UBUNTU_BOX_NAME="ubuntu-trusty-20150204.1" # used in filenames for caching, bump at the same time as UBUNTU_BOX_URL
```
Comment (lmars) [inline script/release-vm-images]: This variable needs bumping to invalidate the cache (we could just use `UBUNTU_BOX_SHA` for this).

File: util/packer/ubuntu-14.04/scripts/upgrade.sh (hunk 1 above)
Comment (titanous) [inline util/packer/ubuntu-14.04/scripts/upgrade.sh]: I'm not sure I follow? The `dist-upgrade` below can still result in a new kernel.

File: script/release-vm-images (hunk 2 above)
Comment (titanous) [inline script/release-vm-images]: Changed to use the SHA.

File: util/packer/ubuntu-14.04/scripts/upgrade.sh (hunk 1 above)
Comment (lmars) [inline util/packer/ubuntu-14.04/scripts/upgrade.sh]: Well, I meant just don't upgrade the kernel and use the one from the release image.

**Comment by lmars** [comment]
LGTM

File: util/packer/ubuntu-14.04/scripts/upgrade.sh (hunk 1 above)
Comment (titanous) [inline util/packer/ubuntu-14.04/scripts/upgrade.sh]: Ah, I think we should always update the kernel.

**Pull request merged**